src/catalog/plugin/src/main/java/org/geoserver/config/plugin/GeoServerImpl.java

```diff
@@ -14,16 +14,7 @@
 import org.geoserver.catalog.impl.ModificationProxy;
 import org.geoserver.catalog.impl.ResolvingProxy;
 import org.geoserver.catalog.plugin.Patch;
-import org.geoserver.config.ConfigurationListener;
-import org.geoserver.config.GeoServer;
-import org.geoserver.config.GeoServerFacade;
-import org.geoserver.config.GeoServerFactory;
-import org.geoserver.config.GeoServerInfo;
-import org.geoserver.config.GeoServerLoader;
-import org.geoserver.config.GeoServerLoaderProxy;
-import org.geoserver.config.LoggingInfo;
-import org.geoserver.config.ServiceInfo;
-import org.geoserver.config.SettingsInfo;
+import org.geoserver.config.*;
```

Comment:
no wildcard imports allowed. `make lint` will fail with

```
[INFO] --- fmt:2.21.1:check (default-cli) @ gs-cloud-catalog-plugin ---
[info] Processed 92 files (1 non-complying).
[ERROR] Found 1 non-complying files, failing build
[ERROR] To fix formatting errors, run "mvn com.spotify.fmt:fmt-maven-plugin:format"
[ERROR] Non complying file:  src/catalog/plugin/src/main/java/org/geoserver/config/plugin/GeoServerImpl.java
```

Reply:
Done in 33c46d24e75dd4a50af3d908695a1e739d75ce4b